### PR TITLE
complie on dragonfly

### PR DESCRIPTION
**Pull request by sysadminmike**

### CODE REVIEW

File: Godeps/_workspace/src/github.com/calmh/osext/osext_procfs.go [hunk 1]
```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-// +build linux netbsd openbsd solaris
+// +build linux netbsd openbsd solaris dragonfly
```
Comment (AudriusButkevicius) [inline Godeps/_workspace/src/github.com/calmh/osext/osext_procfs.go]: This actually needs a PR in https://github.com/calmh/osext

**Comment by sysadminmike** [comment]
ok - will try to submit a PR there

**Comment by sysadminmike** [comment]
should i close this one and open a new one for the  cmd/syncthing/memsize_unimpl.go  change?

**Comment by calmh** [comment]
Just rebase this to a single commit on memsize_unimpl, and open the other PR as well (you may already have, I haven't looked), and we're cool.

**Comment by calmh** [comment]
Oh, and the commit subject should be something along the lines of "Update memsize_unimpl.go for dragonfly (fixes #1351)" (the issue number being the main thing).

**Comment by calmh** [comment]
@st-jenkins test this please

**Comment by sysadminmike** [comment]
sorry not sure how todo that in git - is it simpler for me to close this PR and create a new one just for the memsize_unimpl.go change?

**Comment by AudriusButkevicius** [comment]
Just do a forced push to your branch, that will effectively rewrite the PR.
Also, add yourself to authors (in all 3 places, check previous commits to see where)

**Comment by sysadminmike** [comment]
ahh ok - i didnt create a local branch and was working on a copy from the freebsd ports tree and just did it via githubs web editor

**Comment by sysadminmike** [comment]
i think the port version is also a few versions behind the current branch and is on v0.10.12
fyi all running well on dragonfly

**Comment by calmh** [comment]
Added, thanks.

**Comment by sysadminmike** [comment]
nice - thanks

**Comment by calmh** [comment]
@sysadminmike btw also added so that we build dragonfly binaries now, will be in the next release (or http://build.syncthing.net/job/syncthing/ now).

**Pull request closed without merging**